### DISCUSSION TITLE
markers in different files used in macro

### DISCUSSION THREAD

**Discussion by mrJC0** [JetBrains/ideavim#552] [Q&A] [answered]
Hello,

Using 
IdeaVim 1.10.3 + Rider 2022.1.2

I am having trouble, with a simple macro that should paste different text (e.g. boiler plate code) over several files:

1. setup marks  `mA` and `mB` each in different file
2. record and run simple macro as presented in register: `'Aiaa^['Bibb^[` 

I was hoping for the same behaviour i observed during the recording of this macro - that is go to each mark and print text - 'aa' ending up at mark A and 'bb' ending up at mark B.

### Issue when run from file with mark 'A'
- jumps to 'A' mark then print's 'aabb'
### Issue when run from arbitrary file
-  'aabb' end's up on current cursor position

Finally the IDE-file-tab always switches to 'B' mark eventualy, but text is not printed where i would have hoped.

Note that macro work's just fine if both marks are placed in the currently-opened file.
Incidently i got the same behaviour when using script sequence 
`norm 'A`
 `norm itext0`
 `norm 'B`
 `norm itext1` 

<br/><br/>
Sorry for possible confustion, i am no Vim wizard :), and likely i misunderstood the usage of Vim commands in the first place.
<br/>
Thank you for the effort put into this project.

**Comment by AlexPl292** [comment, marked as answer]
Hi, please track the following issue for the updates

**Reply by mrJC0** [reply]
Hello, you maybe wanted to paste the link for the issue?

**Reply by AlexPl292** [reply]
Oh, yes, sorry, it's https://youtrack.jetbrains.com/issue/VIM-1705/Support-moving-between-files-when-recording-macros
